tests/benchmarks/model_benchmarks/test_pytorch_cnn.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
The same comments: we should keep the code coverage for patch is > 80%.

Reply:
Solved.

Reply:
Done. Currently `93.90% of diff hit (target 80.00%)`